### DISCUSSION TITLE
Some issues about font loading

### DISCUSSION THREAD

**Discussion by wangxun2008** [raysan5/raylib#3989] [General]
![image](https://github.com/raysan5/raylib/assets/88886127/07eed89b-409b-4959-b93f-4cb8dda30e13)

Here is my code.

```
#include <raylib.h>
#include <bits/stdc++.h>
using namespace std;

int main() {
	
	InitWindow(800,600,"");
	int fileSize;
	unsigned char *fontFileData = LoadFileData("c:\\windows\\fonts\\simhei.ttf", &fileSize);
	int codepointsCount;
	int *codepoints=LoadCodepoints("head",&codepointsCount);
	Font font = LoadFontFromMemory(".ttf",fontFileData,fileSize,64,codepoints,codepointsCount);
	UnloadCodepoints(codepoints);
	SetTargetFPS(120);
	while (!WindowShouldClose()) {
		BeginDrawing();
		ClearBackground(WHITE);
		DrawTextEx(font,"head",(Vector2){50,50},64,5,RED);
		EndDrawing();
	}
	UnloadFont(font);
	UnloadFileData(fontFileData);
	return 0;
}
```

Some characters seemed to be missing. I found this problem after installing raylib v5.1-dev. This was no problem with version 4.5。
Is there anything wrong with my code?

**Comment by JupiterRider** [comment]
@wangxun2008 I have the same problem with v5.0 as well. 
You may want to open an issue for that.

I think it has to do with the font size (for example 32 instead of 64 works).

**Reply by wangxun2008** [reply]
Yes, it looks right when using other font sizes. But I still hope this bug can be fixed.

**Comment by raysan5** [comment]
@wangxun2008 Could you try drawing `font.texture`? Probably the required codepoint has not been loaded... Also, feel free to open an issue for review.
